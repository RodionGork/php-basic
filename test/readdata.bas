rep:
read x,y
if x < 1 then restore : goto rep2
print "hypot(";x;",";y;")=";sqr(x^2 + y^2)
goto rep
data 3,4,5,12,0,0

rep2:
read a : print a : if a > 0 then goto rep2