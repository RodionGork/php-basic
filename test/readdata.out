hypot(3,4)=5
hypot(5,12)=13
3
4
5
12
0
